webrob/app_and_db.py

```diff
@@ -7,29 +7,52 @@
 # Authors: Ling Thio <ling.thio@gmail.com>
 
 import os
-from flask import Flask
+from flask import Flask, session
 from flask_sqlalchemy import SQLAlchemy
 import logging
 from pymongo import MongoClient
+import pymongo
+
 
-MONGO_HOST = os.environ.get('KNOWROB_MONGO_HOST')
-MONGO_PORT = int(os.environ.get('KNOWROB_MONGO_PORT'))
-MONGO_DB = os.environ.get('KNOWROB_MONGO_DB')
-MONGO_USER = os.environ.get('KNOWROB_MONGO_USER')
-MONGO_PASS = os.environ.get('KNOWROB_MONGO_PASS')
 
 # This is the WSGI compliant web application object
 app = Flask(__name__)
 app.logger.addHandler(logging.StreamHandler())
 app.logger.setLevel(logging.INFO)
 
-# remote mongo client connection
-connection = MongoClient(MONGO_HOST, MONGO_PORT)
-mongoDbClient = connection[MONGO_DB]
-mongoDbClient.authenticate(MONGO_USER, MONGO_PASS)
-mongoDBMetaCollection = mongoDbClient["meta"]
-if mongoDBMetaCollection.count() > 0:
-    app.logger.info('------------ mongoDb collection contains some document values ------------')
+# set up flask SQLAlchemy db uri
+app.config['SQLALCHEMY_DATABASE_URI'] =  'postgresql://docker@' + \
+os.environ['POSTGRES_PORT_5432_TCP_ADDR'] + ':' + \
+os.environ['POSTGRES_PORT_5432_TCP_PORT'] + '/docker'
 
 # This is the SQLAlchemy ORM object
 db = SQLAlchemy(app)
+
+
+# This method will check db connection with given settings.
+# If settings are correct then will return meta collection from the db
+def checkConnection(neemHubSettings):
```

Comment:
the naming is confusing, methods that return some object should have *get* prefix.
prefix *check* suggests to me that the function returns a boolean indicating whether the check succeeds or not.

Reply:
renamed method to getMongoDBMetaCollection